dashboard/src/components/Common/Autocomplete/AutocompleteField.jsx

```diff
@@ -57,6 +60,20 @@ class AutocompleteField extends React.Component {
     })
   }
 
+  handleHover(suggestion) {
+    this.props.fieldProps.onChange(suggestion)
+  }
+
+  tabCheck(event, suggestions, fieldProps) {
```

Comment:
It's a bit subtle, but I think `suggestions` and `fieldProps` are local to this object, so you may just want to look them up from `this` rather than passing them in. `tabCheck` is not meant to be used outside the context of the local component, so we might as well assume those fields are available via `this`.